### PR TITLE
numpydoc-ify SymLogNorm

### PR DESCRIPTION
**Pull request by dstansby**
I have made no attempt to update the text, but this should still be a small improvement.

### CODE REVIEW

**Comment by dstansby** [comment]
I've got into the habit of doing it this way because that's what [numpydoc says to do](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring). Happy to change it though.

**Comment by anntzer** [comment]
Let's see whether @timhoffm has an opinion there :)

File: lib/matplotlib/colors.py [hunk 1]
```diff
@@ -1193,23 +1193,22 @@ class SymLogNorm(Normalize):
     need to have a range around zero that is linear.  The parameter
     *linthresh* allows the user to specify the size of this range
     (-*linthresh*, *linthresh*).
+
+    Parameters
+    ----------
+    linthresh : scalar
```
Comment (timhoffm) [inline lib/matplotlib/colors.py]:
```suggestion
    linthresh : float
```

File: lib/matplotlib/colors.py [hunk 2]
```diff
@@ -1193,23 +1193,22 @@ class SymLogNorm(Normalize):
...
+        The range within which the plot is linear (to avoid having the plot go
+        to infinity around zero).
+    linscale : scalar, optional
```
Comment (timhoffm) [inline lib/matplotlib/colors.py]:
```suggestion
    linscale : float, default: 1
```

File: lib/matplotlib/colors.py [hunk 3]
```diff
@@ -1193,23 +1193,22 @@ class SymLogNorm(Normalize):
...
+        This allows the linear range (-*linthresh* to *linthresh*) to be
+        stretched relative to the logarithmic range. Its value is the number of
+        decades to use for each half of the linear range.  For example, when
+        *linscale* == 1.0 (the default), the space used for the positive and
+        negative halves of the linear range will be equal to one decade in
+        the logarithmic range. Defaults to 1.
```
Comment (timhoffm) [inline lib/matplotlib/colors.py]:
```suggestion
        the logarithmic range.

**Pull request merged**